payara-micro/demystifying-microservices-example/CrudView/src/main/webapp/js/libs/jquery/jquery.js

```diff
@@ -0,0 +1,10220 @@
+/*!
```

Comment:
Is it possible to remove the jQuery and Bootstrap libraries in favour of using a CDN?